src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -74,23 +47,19 @@ internal static void Initialize()
             {
                 Initializing = true;
 
-                if (s_controllerInstance == null)
+                if (!IsControllerInitialized)
                 {
-                    int enabled = Config_EnableEventPipe;
-                    if (enabled > 0)
+                    if (Config_EnableEventPipe > 0)
                     {
                         // Enable tracing immediately.
                         // It will be disabled automatically on shutdown.
                         EventPipe.Enable(BuildConfigFromEnvironment());
                     }
-                    // If not set at all, we listen for changes in the control file.
-                    else if (enabled != 0)
-                    {
-                        // Create a new controller to listen for commands.
-                        s_controllerInstance = new EventPipeController();
-                    }
+
                     // If enable is explicitly set to 0, then don't start the controller (to avoid overhead).
```

Comment:
I don't think this comment is correct anymore since this is now static class and no controller exists anymore.